functions/replace-route/app.py

```diff
@@ -113,59 +116,217 @@ def get_nat_gateway_id(vpc_id, subnet_id):
     return nat_gateway_id
 
 
-def replace_route(route_table_id, nat_gateway_id):
-    new_route_table = {
-        "DestinationCidrBlock": "0.0.0.0/0",
-        "NatGatewayId": nat_gateway_id,
-        "RouteTableId": route_table_id
-    }
+def replace_route(route_table_id, new_route_table):
```

Comment:
Sorry, what I meant was something like this:

```
def replace_route(route_table_id, target_id):
    new_route_table = {}
    target_key = "NatGatewayId"
    if target_id.startswith("eni-"):
        target_key = "InstanceId"
    new_route_table = {
        "DestinationCidrBlock": "0.0.0.0/0",
        "RouteTableId": route_table_id,
        target_key = target_id
    }
```

Then you can call it just as it was before.